.gitignore

```diff
@@ -1,3 +1,4 @@
 modelsim/work/*
+target/*
```

Comment:
Please add a README.md under the target.
Then add "!target.README.md" so it worn be ignored